crates/router/src/services/api/client.rs

```diff
@@ -31,16 +29,16 @@ fn get_client_builder(
                 .unwrap_or_default(),
         ));
 
-    if should_bypass_proxy {
-        return Ok(client_builder);
-    }
+    let proxy_exclusion_config =
+        reqwest::NoProxy::from_string(&proxy_config.bypass_proxy_hosts.clone().unwrap_or_default());
```

Comment:
Using `unwrap_or_default()` here, since `NoProxy::from_string()` returns `None` if the string is empty, and since `Proxy::no_proxy()` accepts `Option<NoProxy>`.